bagitobjecttransfer/recordtransfer/admin.py

```diff
@@ -102,7 +107,7 @@ def has_delete_permission(self, request, obj=None):
 
 @admin.register(TempUploadedFile)
 class TempUploadedFileAdmin(ReadOnlyAdmin):
```

Comment:
You can reuse this admin for both models:

```suggestion
@admin.register(TempUploadedFile)
@admin.register(PermUploadedFile)
class UploadedFileAdmin(ReadOnlyAdmin):
```

Reply:
Just a slight syntax adjustment since stacking decorators like that does not work.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/337/commits/deccec1809d2bcebd9329ef47b60670dea794fac